.github/ISSUE_TEMPLATE/1.bug_report.yaml

```diff
@@ -0,0 +1,146 @@
+name: Bug report
+description: Create a report to help us improve Amplify Codegen
+title: ""
+issue_body: true
+body:
+- type: markdown
+  attributes:
+    value: Thanks for taking the time to fill out this bug report! Try to include as much information as you can. 
+- type: markdown
+  attributes:
+    value: For reference, the current version of the Amplify CLI is <img src="https://img.shields.io/npm/v/amplify-codegen.svg" />.
+- type: markdown
+  attributes:
+    value: |
+      > **Note**: If your bug does not result from running `amplify codegen <sub-command>`, please log it in the [Amplify CLI GitHub Issue Tracker](https://github.com/aws-amplify/amplify-cli/issues).
+- type: checkboxes
+  attributes:
+    label: |
+      Before opening, please confirm:
+    options:
+      - label: I have installed the latest version of the Amplify CLI (see above), and confirmed that the issue still persists.
```

Comment:
```suggestion
      - label: I have installed the latest version of the Amplify Codegen (see above), and confirmed that the issue still persists.
```

Reply:
Our Codegen plugin is currently installed as part of the Amplify CLI. The codegen version that is be installed as part of CLI will be fixed major, latest minor and patch version. Currently we do not provide a command like `amplify codegen -v` for the customers to report this version. Happy to come back and change this when we add this feature.